### ISSUE TITLE
[Revise subclass] isolated congenital microcephaly

### ISSUE THREAD

**Issue by pnrobinson** [monarch-initiative/mondo#5683]
**Mondo term (ID and Label)**
[isolated congenital microcephaly](https://www.ebi.ac.uk/ols/ontologies/mondo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0016056&lang=en&viewMode=PreferredRoots&siblings=false)

**Suggested revision and reasons**

All of the descendents are Mendelian and I think we need to define this term as Mendelian and place it under a (partially revised) neurogenetic hierarchy. @ahamosh  thoughts?

**Comment by ahamosh** [comment]
I agree.  I am confused by all the parents this has.  Congenital microcephaly is different from acquired microcephaly.  Isolated is different from syndromic.  Let's discuss tomorrow.

**Comment by nicolevasilevsky** [comment]
microcephaly is usually described as primary or acquired - primary is where you are born microcephalic and acquired is you are born with a normal head size but it eventually is relatively small (does not grow)

we have
'autosomal dominant primary microcephaly' and 'autosomal recessive primary microcephaly'- which are hereditary

should 'isolated microcephaly' be in HPO rather than Mondo?

it makes sense to classify 'isolated congenital microcephaly' as Mendelian. May want to add hereditary to the label.

**Comment by pnrobinson** [comment]
You can have microcephaly from congenital CNV, the feature is not always Mendelian. However, we would not call the disease "CNV microcephaly" but rather "congenital CNV" and the microcephaly is a feature. 

We usually say primary or secondary microcephaly, I think, not "acquired".
I believe all of the diseases that we call "<something> microcephaly are Mendelian (although microcephaly is a feature of many other diseases). My suggestion is the change the name of the term from

isolated congenital microcephaly 

to 

Familial microcephaly

and to put this under a neurogenetic branch. 

@ahamosh  does this sound good?

**Comment by ahamosh** [comment]
Can we make it hereditary microcephaly instead of familial microcephaly.  People are moving away from familial because of de novo variants.
